data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -312,6 +315,7 @@ then
 			--haproxy-image="${HAPROXY_IMAGE}" \
```

Comment:
In theory we could also drop these arguments now, right?

Reply:
We could, yes, but I was a big chicken :smile: 

I figured we'd add the new argument -> make sure on our end everything works (I'm sure it does)  -> come back for cleanup.  If you think we should pull the old args out now to save time, we can?

Reply:
We wouldn't pass CI here unless the new path worked entirely, right?  (I guess there's possible corner cases on e.g. bare metal images not used in the default cloud cases, but we can probably just double check before merging)

I don't have a strong opinion here and will leave it to your choice!